history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -1463,6 +1603,10 @@ which was held this summer at London, again to draw up an address to the king,
 and to acquaint him of one thing more, which continued to be troublesome to them.
 This address they presented to him, and was as follows:
 
+[.embedded-content-document.address]
+--
+
+[.letter-heading]
 To King James II. over England, etc.
 
 The humble address of the People called Quakers, from their Yearly-Meeting in London,
```

Comment:
Should this be [ssco]?

Reply:
I don't think so.  It makes me wish we had something like `[.letter-sub-heading]` -- but we don't.  Maybe I'll add one.  But let's just leave it like this for now.